sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/Generated/ConfidentialLedgerClient.cs

```diff
@@ -54,6 +54,25 @@ public ConfidentialLedgerClient(Uri ledgerUri, TokenCredential credential, Confi
         }
 
         /// <summary> The constitution is a script that assesses and applies proposals from consortium members. </summary>
+        /// <remarks>
+        /// Schema for <c>Response Body</c>:
```

Comment:
nit: Should this be `<c>Response.Content</c>` or `<see cref="Response.Content"/>`

Reply:
I could see the argument for doing so.  One possible issue is that in a case where you use `RequestOptions` to prevent a `RequestFailedException` from being thrown in the case where the operation fails, `Response.Content` is going to be the content the server returned for the error, which won't match this schema.

Maybe keep it as is but if we discover lots of people have problems discovering the `Content` property on `Response` we can see some combination of `see` or `seealso` to point folks at the right thing?

Reply:
My main concern is that there is no concept of `Body` on the Response type.

Reply:
I'll do a PR to the code generator to replace "Content" with "Body" here, but I'll hold off on making it a `see` for now.  Do you think that has a ok balance for now, @christothes?